src/libraries/Microsoft.Extensions.Configuration/src/ChainedConfigurationProvider.cs

```diff
@@ -36,8 +36,8 @@ public ChainedConfigurationProvider(ChainedConfigurationSource source)
         /// Tries to get a configuration value for the specified key.
         /// </summary>
         /// <param name="key">The key.</param>
-        /// <param name="value">The value.</param>
-        /// <returns><c>True</c> if a value for the specified key was found, otherwise <c>false</c>.</returns>
+        /// <param name="value">When this method returns, contains the value.</param>
+        /// <returns><c>true</c> if a value for the specified key was found, otherwise <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> if a value for the specified key was found, otherwise <see langword="false"/>.</returns>
```